lib/querystring.js

```diff
@@ -76,7 +76,12 @@ const unhexTable = new Int8Array([
   -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
   -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,  // ... 255
 ]);
-// A safe fast alternative to decodeURIComponent
+/**
+ * A safe fast alternative to decodeURIComponent
+ * @param {any} s
```

Comment:
why not `string` ?

Reply:
```suggestion
 * @param {string} s
```

reading the code it does expect a string due to usage of `StringPrototypeCharCodeAt`